meetuppizza/tests.py

```diff
@@ -13,6 +19,49 @@
     }
 
 class TestLandingPage(TestCase):
+  def setUp(self):
+    self.meetup_info = {
+          0: {
+          "created": 1426723243000,
+          "duration": 5400000,
+          "group": {
+            "created": 1391476627000,
+            "name": "Software Craftsmanship New York",
+            "id": 12705402,
+            "join_mode": "approval",
+            "lat": 40.7599983215332,
+            "lon": -73.98999786376953,
+            "urlname": "Software-Craftsmanship-New-York",
+            "who": "craftsmen"
+          },
+          "id": "ldxfglyvfbfc",
+          "link": "http://www.meetup.com/Software-Craftsmanship-New-York/events/229387692/",
+          "name": "Code & Coffee",
+          "status": "upcoming",
+          "time": 1458730800000,
+          "updated": 1453163859000,
+          "utc_offset": -14400000,
+          "yes_rsvp_count": 1,
+          "waitlist_count": 0,
+          "description": "<p>Do you like getting up early and starting the day with inspiring conversations or even better: pair programming on your pet project? This is your meetup! :) </p> <p>Skip the morning run and instead meet us between 6:30 AM and 8:30 AM bi-weekly to chat, code and have the first coffee together to start the day off with developers or other software professionals who care about the quality of the code and passionately looking for ways to improve themselves. </p> <p><b>The venue:</b> There is a Starbucks open from 5:30 am and behind it the lobby of The Lexington NYC hotel, which is a pretty cool space, they have free, fast wi-fi, plenty of seating and a lot of sockets in the wall :) I asked, and we are allowed to gather so don't be shy and have a look around. There are seats downstairs and on the mezzanine as well, upstairs, where the restrooms are too. See you there! </p> ",
+            "venue": {
+              "id": 23708903,
+              "name": "The Lexington",
+              "lat": 40.75501251220703,
+              "lon": -73.97337341308594,
+              "address_1": "511 Lexington Ave",
+              "city": "New York",
+              "country": "us",
+              "localized_country_name": "USA",
+              "zip": "",
+              "state": "NY"
+            },
+          }
+        }
+    self.patcher = patch('meetup.services.meetup_api_lookup_agent.MeetupApiLookupAgent')
+    self.mock_agent = self.patcher.start()
+    self.mock_agent.return_value.get_response.return_value.json.return_value = self.meetup_info
+    self.addCleanup(self.patcher.stop)
```

Comment:
This can go in a `tearDown` method, too. To me that reads a little more clearly.